dsaps/helpers.py

```diff
@@ -25,6 +25,16 @@ def create_csv_from_list(list_name, output):
             writer.writerow([item])
 
 
+def get_bitstreams_from_csv(metadata_csv):
+    bitstreams = {}
+    with open(metadata_csv, "r") as csvfile:
+        reader = csv.DictReader(csvfile)
+        for row in reader:
+            if bitstream := row.get("bitstreams"):
+                bitstreams[row["item_identifier"]] = ast.literal_eval(bitstream)
+    return dict(sorted(bitstreams.items()))
```

Comment:
I'm not seeing any references to this method now, is it officially phased out?

Reply:
Good catch! Removing. 😅